### PR TITLE
Enabled to take a backup of slow_logs

### PR DESCRIPTION
**Pull request by yoheimuta**
It allows to take a backup of slow_log records before the procedure deletes the records.

I want to use this nice plugin as one only to get records, not to delete records, because using sql queries to the table is useful for quick understanding and survey.

Please review PR, and merge If there is no problem.

### CODE REVIEW

**Comment by yoheimuta** [comment]
Sorry, I fixed a test...

File: README.md [hunk 1]
```diff
@@ -26,6 +27,7 @@ every 10 seconds from AWS RDS.
   host [RDS Hostname]
   username [RDS Username]
   password [RDS Password]
+  custom_table [Your Backup Tablename]
```
Comment (winebarrel) [inline README.md]: I think that there is better to change the parameter name
e.g.) `backup_table`

File: lib/fluent/plugin/in_rds_slowlog.rb [hunk 2]
```diff
@@ -70,6 +71,11 @@ def output
       row.each_key {|key| row[key].force_encoding(Encoding::ASCII_8BIT) if row[key].is_a?(String)}
       router.emit(tag, Fluent::Engine.now, row)
     end
+
+    if @custom_table
+      @client.query("CREATE TABLE IF NOT EXISTS #{@custom_table} LIKE slow_log")
```
Comment (winebarrel) [inline lib/fluent/plugin/in_rds_slowlog.rb]: I think it is better to create create a table in `start`

**Comment by yoheimuta** [comment]
Thank you for the review and I fixed the problems.

File: test/plugin/test_in_rds_slowlog.rb [hunk 3]
```diff
@@ -87,6 +88,7 @@ def setup
     username test_rds_user
     password test_rds_password
     interval 0
+    backup_table mysql.slow_log_custom_backup
```
Comment (winebarrel) [inline test/plugin/test_in_rds_slowlog.rb]: I think it is better not to add `backup_table` to the default setting.
Would you like to add `backup_table` in `test_backup`?

```ruby
d = create_driver(CONFIG + 'backup_table mysql.slow_log_custom_backup')
```

**Comment by winebarrel** [comment]
LGTM with nits. :smiley: 

@kenjiskywalker Do you have any thoughts?

**Comment by kenjiskywalker** [comment]
@yoheimuta  @winebarrel 

It seems good to me.
I'll merge this p-r. Thanks!

**Comment by kenjiskywalker** [comment]
Released v0.0.7 :rocket:

**Comment by winebarrel** [comment]
:+1:

**Comment by yoheimuta** [comment]
Thanks ! :smile:

**Pull request merged**